samples/spm/README.rst

```diff
@@ -8,10 +8,10 @@ Secure Partition Manager
    :depth: 2
 
 The Secure Partition Manager sample provides a reference use of the System Protection Unit peripheral.
```

Comment:
Should the section Automatic building of SPM in this file be completely removed now?

Reply:
Adding "when SPM is enabled" would be a better solution.